hw/top_earlgrey/data/chip_conn_testplan.hjson

```diff
@@ -1392,5 +1392,16 @@
               "ast_cc2"]
       tags: ["conn"]
     }
+
+    /////////////////////////////////
+    // flash_ctrl_nvm_debug_en.csv //
+    /////////////////////////////////
+    {
+      name: flash_ctrl_nvm_debug
+      desc: "Verify the connectivity between the lc_ctrl output and the flash_ctrl input."
+      stage: V2
+      tests: ["flash_ctrl_nvm_debug_en"]
```

Comment:
If we're only testing this particular connection, we don't have to do anything and can close this PR ;).
Search for `lc_nvm_debug_en_flash_ctrl` in this file...